### PR TITLE
added local disconnection in GCA

### PR DESCRIPTION
**Pull request by n1mh**

### CODE REVIEW

File: data/glade/gecoscon.glade [hunk 1]
```diff
@@ -82,7 +82,7 @@
                         <property name="visible">True</property>
                         <property name="can_focus">False</property>
                         <property name="xalign">0.69999998807907104</property>
-                        <property name="pixbuf">../media/logo-gecos.png</property>
+                        <property name="pixbuf">../media/btn-update.png</property>
```
Comment (alfem) [inline data/glade/gecoscon.glade]: Replacing this GECOS logo with a btn-update.png seems like a mistake to me.

Other buttons are being replaced too. Is there a reason?

File: data/glade/gecoscon.glade [hunk 2]
```diff
@@ -562,7 +577,7 @@
                               <object class="GtkImage" id="statusicon">
                                 <property name="visible">True</property>
                                 <property name="can_focus">False</property>
-                                <property name="pixbuf">../media/btn-status.png</property>
+                                <property name="pixbuf">../media/btn-update.png</property>
```
Comment (alfem) [inline data/glade/gecoscon.glade]: Another button icon replaced

File: data/glade/gecoscon.glade [hunk 3]
```diff
@@ -605,7 +620,7 @@
                               <object class="GtkImage" id="usersicon">
                                 <property name="visible">True</property>
                                 <property name="can_focus">False</property>
-                                <property name="pixbuf">../media/btn-users.png</property>
+                                <property name="pixbuf">../media/btn-update.png</property>
```
Comment (alfem) [inline data/glade/gecoscon.glade]: Another button replaced

File: data/glade/gecoscon.glade [hunk 4]
```diff
@@ -648,7 +663,7 @@
                               <object class="GtkImage" id="managericon">
                                 <property name="visible">True</property>
                                 <property name="can_focus">False</property>
-                                <property name="pixbuf">../media/btn-manager.png</property>
+                                <property name="pixbuf">../media/btn-update.png</property>
```
Comment (alfem) [inline data/glade/gecoscon.glade]: Another button replaced

File: data/glade/gecoscon.glade (hunk 1 above)
Comment (n1mh) [reply data/glade/gecoscon.glade]: There's no reason. Glade editor changed that images. I've restored them all.

**Pull request merged**